tests/Paysera/WalletApi/Entity/ClientPermissionsToWalletTest.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+use Paysera\WalletApi\WalletTrait;
+
+class Paysera_WalletApi_Entity_ClientPermissionsToWalletTest extends PHPUnit_Framework_TestCase
```

Comment:
We don't write tests to Entitites, we cover entity with the tests in services, so it's ok to cover with the tests Mapper, Manager and similar things.

Reply:
May I leave this test? It's too pity to delete it :(

Reply:
Done